### PR TITLE
traduccion de alternatives.md

### PR DESCRIPTION
**Pull request by jorgerpo**
**traduccion de alternatives.md**

### CODE REVIEW

**Comment by mariacamilagl** [comment]
Hi @jorgerpo, thank you for your contribution.  ☕ 🌮 

I was just seeing that the translation is not finished, right? ⚙️ 

The next steps would be then to finish translating the entire file and erasing the extra files that seem to have been included in git. 🚀 

If you need further guidelines about the style of the translation you can refer to the original issue 😃  https://github.com/tiangolo/fastapi/issues/1218

**Comment by jorgerpo** [comment]
Hello Camila
In my mind is was finished.
I'll check
I think that maybe I didn't run the markdown parser to verify how it outputs the html

for instance, when it says **!!!check**
I wrote **!!!mira**

And after sending the pull request I realized that wasn't meant to be translated.

But I am very open to check and finish if you please point out what is still undone.

**Comment by jorgerpo** [comment]
imagen
para clarificar, lo que traduje fue alternatives.md
No sé como verificar que el markdown esté procesando bien
en la imagen se ve que donde decía !!! check yo puse !!! mira
y donde decía !!! note , puse !!! nota

creo que es lo único que falta

si me ayudas a verifcar eso, lo dejo listo y podría seguir traduciendo otras páginas
y perdon la primiparada en estos temas
gracias

File: docs/es/docs/alternatives.md [hunk 1]
```diff
@@ -0,0 +1,424 @@
...
+
+Por ser tan  simple, es relativamente intuitivo de aprender, aunque la documentación se vuelve algo técnica en algunos apartes.
+
+También se utiliza comúnmente para otras aplicaciones que no necesariamente requieran  bases de datos, gestión de usuarios, o cualquiera de las tantas características que vienen por defecto en Django. Aunque, vale aclarar, muchas de estas características se pueden agregar con complementos.
+
+Este desacoplamiento de sus partes, y el hecho de ser un "microframework" que se puede  extender para cubrir exactamente lo que se requiere, era una característica clave que yo quería conservar.
+
+Dada la simplicidad de Flask, parecía una buena combinación para construir API. Ahora tenía que hallar un "Django REST Framework" para Flask.
+
+Given the simplicity of Flask, it seemed like a good match for building APIs. The next thing to find was a "Django REST Framework" for Flask.
```
Comment (mariacamilagl) [inline docs/es/docs/alternatives.md:59]: Hi @jorgerpo, thank you for your super fast reply!  🚀 

As you can check here, from this line forward this file is in English.

File: docs/es/docs/benchmarks.md [hunk 2]
```diff
@@ -0,0 +1,34 @@
+# Benchmarks
```
Comment (mariacamilagl) [inline docs/es/docs/benchmarks.md:1]: Please remove this file. 😄

File: docs/es/docs/deployment.md [hunk 3]
```diff
@@ -0,0 +1,394 @@
+# Deployment
```
Comment (mariacamilagl) [inline docs/es/docs/deployment.md:1]: Please remove this file. 😄

File: docs/es/docs/project-generation.md [hunk 4]
```diff
@@ -0,0 +1,102 @@
+# Project Generation - Template
```
Comment (mariacamilagl) [inline docs/es/docs/project-generation.md:1]: Please remove this file. 😄

File: docs/es/docs/release-notes.md [hunk 5]
```diff
@@ -0,0 +1,805 @@
+# Release Notes
```
Comment (mariacamilagl) [inline docs/es/docs/release-notes.md:1]: Please remove this file. 😄

**Review by mariacamilagl — CHANGES_REQUESTED** [review]
Thank you for your contribution @jorgerpo ! 🌮 

I added some comments here so you can check what's missing in the translation and the files that should be removed.

Don't worry, we were all beginners once 😃 Your effort is appreciated! 🚀

**Comment by tiangolo** [comment]
I'm gonna close this one for now, to clean up stale PRs. But if you get the chance to fix those things at some point later, feel free to create a new PR! Thanks for the effort. ☕

**Pull request closed without merging**